config/webpack.config.js

```diff
@@ -365,12 +365,7 @@ module.exports = function (webpackEnv) {
             // A missing `test` is equivalent to a match.
             {
               test: [/\.bmp$/, /\.gif$/, /\.jpe?g$/, /\.png$/],
-              type: 'asset',
-              parser: {
-                dataUrlCondition: {
-                  maxSize: imageInlineSizeLimit,
-                },
-              },
+              type: 'asset/resource',
```

Comment:
I change this because the `type: 'asset',` is having problems with png file

reference: https://webpack.js.org/guides/asset-modules/#resource-assets